components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
```

Comment:
"its just a warning" is also bad, since people will tend to accumulate them and at some point we will not be able to see errors in a glance.
Or just not event take care of them

So, I think (outside of this pr) we fix the linter rules, we make that specific line not trigger a warning or we came up with another solution

I think letting the lint add up a lot of warnings will tend users to not care about them.
And disable for the full file will have it's problems too as Beto said